src/webgpu/api/operation/rendering/stencil.spec.ts

```diff
@@ -16,6 +16,8 @@ type TestStates = {
   state: GPUDepthStencilState;
   color: Float32Array;
   stencil: number;
+  depth?: number;
+  setStencilReference?: boolean;
```

Comment:
nit: How about merging stencil and setStencilReference? requiring `stencil` when it's not used is confusing to read
```suggestion
  stencil: number | 'unset';
  depth?: number;
```

Reply:
Ah, I see. Then, let me use `stencil: number | undefined` instead.